controllers/authorizeService.js

```diff
@@ -0,0 +1,120 @@
+var AccountingService = require('../db/schemas/accountingService'),
+    config = require('../config'),
+    uuid = require('node-uuid');
+
+var authorizeService = (function () {
+
+    /**
+     * Generates am aìKey.
+     */
+    var generateApiKey = function (callback) {
+        var apiKey = uuid.v4();
+
+        return callback(apiKey);
+    };
+
+    /**
+     * Check if the remote cliente is the WStore; otherwise return an error.
+     */
+    var checkRemoteClient = function (req, callback) {
```

Comment:
Do not use callbacks when not required! You can return `true` or `false` so the code will be easier to understand.